src/app/components/__tests__/__snapshots__/HomePage.spec.tsx.snap

```diff
@@ -4,19 +4,27 @@ exports[`HomePage component should render 1`] = `
 <DocumentFragment>
   <main>
     <div
-      class="hero-header"
+      class="hero-header home-page__header"
     >
       <div
         class="hero-header__content"
       >
         <h1>
-          Find your protein
+          Find your 
+          <button
```

Comment:
Not sure this should be a button? I wouldn't expect it to be clickable and the action triggered is far from it.

Reply:
I understand this is subjective though, so if you feel strongly about it we could run it past the team "what would you expect to happen if you clicked on the text?"

Reply:
Not sure either. I mainly made it a button because otherwise eslint would complain that this breaks 2 accessibility linting rules. We can either leave it as a button, or make it just plain inaccessible on purpose and override the rules here because the action it's covering is already handled by the actual dropdown.

Reply:
I would go for option 2 - as you say it's already covered